scripts/train/train.py

```diff
@@ -150,6 +151,18 @@ def build_composer_model(model_cfg: DictConfig,
 
 
 def main(cfg: DictConfig) -> Trainer:
+    # Run user provided code if specified
+    code_paths = pop_config(cfg,
+                            'code_paths',
+                            must_exist=False,
+                            default_value=[],
+                            convert=True)
+    for code_path in code_paths:
+        try:
+            import_file(code_path)
+        except Exception as e:
+            raise ValueError(f'Error importing code from {code_path}: {e}')
```

Comment:
This adds custom registry stuff?

Reply:
yeah, it allows you to inject your code that registers stuff